client/proto/state_snapshot/v1/state_snapshot.proto

```diff
@@ -0,0 +1,78 @@
+// Copyright 2021 VMware, all rights reserved
+//
+// Concord Client's State Snapshot Service
+
+syntax = "proto3";
+package vmware.concord.client.snapshot.v1;
```

Comment:
File path and package name should be equivalent. I like `snapshot` due to the context should be clear but I'm not opposed to `state_snapshot` either.

Reply:
Good catch, fill fix.